### PR TITLE
Give PMC SG a mag harness

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

Title.

## Why It's Good For The Game

When players spawn in as PMC ERT and get PMC SG, apparently they don't have a mag harness. As par usual of SG gameplay, you don't want to lose that smartweapon. It's inside backpack.

## Changelog
:cl:
fix: PMC SG now has a mag harness in backpack
/:cl:

### CODE REVIEW

**Comment by novaepee** [comment]
opps, deleted branch by accident

File: code/datums/jobs/job/pmc.dm [hunk 1]
```diff
@@ -92,6 +92,7 @@
 	H.equip_to_slot_or_del(new /obj/item/storage/pill_bottle/tricordrazine, SLOT_IN_BACKPACK)
 	H.equip_to_slot_or_del(new /obj/item/ammo_magazine/standard_smartmachinegun, SLOT_IN_BACKPACK)
 	H.equip_to_slot_or_del(new /obj/item/ammo_magazine/standard_smartmachinegun, SLOT_IN_BACKPACK)
+	H.equip_to_slot_or_del(new /obj/item/attachable/magnetic_harness, SLOT_IN_BACKPACK)
```
Comment (BraveMole) [inline code/datums/jobs/job/pmc.dm:95]: it's silly to not do it for them

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/datums/jobs/job/pmc.dm (hunk 1 above)
Comment (novaepee) [reply code/datums/jobs/job/pmc.dm:95]: Let me figure out how to put mag harness on a preset T-29. I might have an idea.

**Comment by novaepee** [comment]
agh, my branch got deleted. time to make new pr

**Pull request closed without merging**